ProjectCoimbra.UWP/Project.Coimbra/Strings/de-DE/Resources.resw

```diff
@@ -329,4 +329,52 @@
   <data name="LaneSetupControl.KeysSelected" xml:space="preserve">
     <value>Keys {0} selected.</value>
   </data>
-</root>
+  <data name="CreateAccountPage.Error" xml:space="preserve">
+    <value>This nickname already exists. Please try again.</value>
+    <comment>Error message displayed if nickname already exists when creating account</comment>
+  </data>
+  <data name="LoginPage.Error" xml:space="preserve">
+    <value>Nickname does not exist. Please signup via link below or try again.</value>
+    <comment>Error message displayed when login nickname does not exist</comment>
+  </data>
+  <data name="CreateAccountPage.Coimbra" xml:space="preserve">
+    <value>COIMBRA</value>
+    <comment>Coimbra text on signup page</comment>
+  </data>
+  <data name="CreateAccountPage.CreateAccount" xml:space="preserve">
+    <value>Create account</value>
+    <comment>Create Account title text on signup page</comment>
+  </data>
+  <data name="CreateAccountPage.EnterNickname" xml:space="preserve">
+    <value>Enter a new nickname:</value>
+    <comment>Message on signup page to ask users to enter new nickname</comment>
+  </data>
+  <data name="CreateAccountPage.LinkToLoginPage" xml:space="preserve">
+    <value>Go back to login page? Click here!</value>
```

Comment:
Not a big issue and we don't have to align with any guidance here, but `sign in` is usually preferable for modern applications